add readme (#1)
DevExpress/web-dashboard-demo-preview: README.md

# Dashboard Angular Application

> The following application is an Angular application with the DevExpress Web Dashboard Angular Component inside. 

Reviewer: ```suggestion
This is an Angular application that contains the DevExpress Web Dashboard Angular component. 
```

> The Web Dashboard can be rendered on the client-side, while having data fed to it by an ASP.NET Core / ASP.NET MVC back-end. This means you can:

Reviewer: You say "can be rendered" here, which implies that the Web Dashboard can also be rendered on the server side. Is this correct?

Author: No, the sentence is an opposition for another approach, where Web Dashboard runs as a single application, that contains both client and server sides.

Reviewer: Okay, please make the following changes:

```suggestion
The Web Dashboard can be rendered on the client side, with data supplied by an ASP.NET Core / ASP.NET MVC backend. This allows you to do the following:
```

Author: See https://github.com/DevExpress/web-dashboard-demo-preview/pull/2/files

> - Publish Web Dashboard as a part of applications based on Angular.

Reviewer: ```suggestion
- Publish a Web Dashboard as part of an Angular application.
```

> - Integrate a dashboard into any place that supports HTTP requests (e.g., see the [GitHub Pages Demo](https://devexpress.github.io/web-dashboard-demo/)).

Reviewer: "any place" is a very broad term. Can you clarify and make it more specific?

Author: It is really any place. The only requirement is to support HTTP requests. It can be sites, applications and so on. I cannot be bore specific here, I do not know where a customer wants to embed Web Dashboard.

Reviewer: ```suggestion
- Integrate a Dashboard into anything that supports HTTP requests (e.g., websites, web applications). See the following demo for details:  [DevExpress Dashboard Demo](https://devexpress.github.io/web-dashboard-demo/)).
```

> - Use the Dashboard side-by-side with [DevExtreme](https://js.devexpress.com) client-side widgets to create highly responsive web apps.

Reviewer: I'd like to rewrite this sentence. For my understanding, do you mean that you can use the _Web_ Dashboard component with DevExtreme client-side widgets?

Author: You can use Web Dashboard and DevExtreme widgets in a single application, connect them with each other.

Reviewer: ```suggestion
- Use the Dashboard with [DevExtreme](https://js.devexpress.com) client-side widgets to create responsive web apps.
```

> The Dashboard Angular application is hosted on the [GitHub Pages](https://pages.github.com), while the server side - on [DevExpress.com](https://www.devexpress.com).

Reviewer: I think we need to clarify what is hosted on the server side here. For example, do you mean:

`In the following demo, the Dashboard Angular application is hosted on [GitHub](https://pages.github.com) and the server-side Dashboard component is hosted on [DevExpress.com](https://www.devexpress.com).`

`- https://devexpress.github.io/web-dashboard-demo/`

Author: ```suggestion
In the following demo, the Dashboard Angular application is hosted on [GitHub](https://pages.github.com) and the server side is hosted on [DevExpress.com](https://www.devexpress.com).
```

Reviewer: "...and the server side is hosted on [DevExpress.com](https://www.devexpress.com)."

Can you clarify, for my understanding, what is on the server side? Is it the underlying database for the dashboard? If so, do we need to specify this?

> **See in action:** https://devexpress.github.io/web-dashboard-demo/

Reviewer: Please delete this line since I connected it to the sentence above.

Author: ```suggestion
**Demo:** https://devexpress.github.io/web-dashboard-demo/
```

Reviewer: Didn't we add "See the following demo for details:  [DevExpress Dashboard Demo]" to the above paragraph? Is it necessary to say this again?

![html-dashboard-demo](https://user-images.githubusercontent.com/17986517/35871357-13796742-0b75-11e8-9b12-a90de4813b96.png)

> > Web Dashboard has components both for [Angular](https://docs.devexpress.com/Dashboard/400409) and [React](https://docs.devexpress.com/Dashboard/400683), but you can also create a pure [HTML JavaScript Web Dashboard](https://github.com/devexpress/web-dashboard-demo) application.

Reviewer: ```suggestion
> Although Web Dashboard components are available for both [Angular](https://docs.devexpress.com/Dashboard/400409) and [React](https://docs.devexpress.com/Dashboard/400683), you can also create an [HTML JavaScript Web Dashboard](https://github.com/devexpress/web-dashboard-demo) application.
```

## Demo Structure

> The application uses the following DevExpress npm packages:

Reviewer: ```suggestion
This demo application uses the following DevExpress npm packages:
```

- `devexpress-dashboard`
- `devexpress-dashboard-angular`
- `@devexpress/analytics-core`
- `devextreme devextreme-angular`

> When you create your own application with the Dashboard Angular component, make sure the following requirements are met:

Reviewer: ```suggestion
When you create your own application with a Dashboard Angular component, ensure that the following requirements are met:
```

> - The script version on the client should match with libraries version on the server up to a minor version.

Reviewer: I'm not sure what you mean by "up to a minor version". Do you mean that the minor version needs to match as well? For example, is it technically correct to say:

```suggestion
- The script version on the client side should match the library version (including the minor version) on the server side.
```

> - Versions of the DevExpress npm packages should be identical (their major and minor versions should be the same).

Reviewer: ```suggestion
- The DevExpress npm package versions should be identical (major and minor versions should be the same).
```

> The client part is an Angular application, where the `DxDashboardControlModule` module is used. Take a look at the following files:

Reviewer: ```suggestion
The client portion is an Angular application, where the `DxDashboardControlModule` module is used. Review the following files:
```

- [app.module.ts](./src/app/app.module.ts)

  Imports the `DxDashboardControlModule` module.
- [app.component.html](./src/app/app.component.html)

  Embeds the `dx-dashboard-control` component.
- [app.component.ts](./src/app/app.component.ts)

  Contains the application logic.
  

>  The server-side of this project is an ASP.NET Core application hosted on the DevExpress.com.

Reviewer: ```suggestion
 The server side of this project is an ASP.NET Core application hosted on DevExpress.com.
```
